src/objectmap-utils/rollup.config.js

```diff
@@ -16,7 +16,7 @@ export default [
       {
         file: packageJson.main,
         format: 'cjs',
-        name: 'SessionDataMapper',
+        name: 'ObjectMap',
```

Comment:
Change to 'ObjectMapUtils' so it doesn't conflict with the jsonforms component if both are used as a CDN